src/game_display.cpp

```diff
@@ -664,3 +676,98 @@ display::overlay_map& game_display::get_overlays()
 {
 	return overlay_map_;
 }
+
+std::vector<texture> game_display::get_reachmap_images(const map_location& loc, image::TYPE image_type)
+{
+	std::vector<std::string> names;
+	const auto adjacent = get_adjacent_tiles(loc);
+
+	enum visibility { REACH = 0, CLEAR = 1 };
+	std::array<visibility, 6> tiles;
+
+	const std::array image_prefix{&game_config::reach_map_prefix};
```

Comment:
Does this need to be an array? Looking at the code, it seems it would crash when accessing `image_prefix[CLEAR]`.